ridgeback_base/include/ridgeback_base/ridgeback_passive_joint.h

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
[Convention](http://wiki.ros.org/CppStyleGuide#Files) for single class files would call this `ridgeback_base/passive_joint_publisher.h`.